packages/notifi-react-card/lib/hooks/index.ts

```diff
@@ -1,3 +1,2 @@
-export * from './useNotifiSubscribe';
```

Comment:
was this intentional?

Reply:
oops! it was throwing an error but it seems to be gone now. updated